packages/cli/package.json

```
@@ -39,8 +39,8 @@
     "url": "https://github.com/stoplightio/prism.git"
   },
   "scripts": {
-    "cli": "node -r ts-node/register -r tsconfig-paths/register src/index.ts",
-    "cli:debug": "node -r ts-node/register -r tsconfig-paths/register --inspect-brk src/index.ts"
+    "cli": "node -r ts-node/register -r ts-node/register/transpile-only src/index.ts",
+    "cli:debug": "node -r ts-node/register -r ts-node/register/transpile-only --inspect-brk src/index.ts"
```

Why is that double indirection in there in the first place?

```suggestion
    "cli": "node -r ts-node/register/transpile-only src/index.ts",
    "cli:debug": "node -r ts-node/register/transpile-only --inspect-brk src/index.ts"
```

This is probably even faster.